Doc/library/typing.rst

```diff
@@ -1949,7 +2205,7 @@ Generic concrete collections
 Corresponding to built-in types
 """""""""""""""""""""""""""""""
 
-.. class:: Dict(dict, MutableMapping[KT, VT])
+.. class:: Dict[KT, VT](dict, MutableMapping[KT, VT])
 
    A generic version of :class:`dict`.
```

Comment:
Hmm, it doesn't necessarily have to be fixed in this PR, but this really isn't a good description anymore, given that `dict` is now generic. Goes for all the other deprecated typing aliases as well.....

Reply:
Let's fix that another day :)